src/main/kotlin/snyk/container/YAMLImageExtractor.kt

```diff
@@ -39,7 +39,7 @@ object YAMLImageExtractor {
     fun extractFromFile(file: VirtualFile, project: Project): Set<KubernetesWorkloadImage> {
         val psiFile = findPsiFileIgnoringExceptions(file, project)
         if (psiFile == null || file.isDirectory || !file.isValid) {
-            logger.warn("no psi file found for ${file.path}")
+            logger.debug("no psi file found for ${file.path}")
```

Comment:
why do we need that event in log?
again, it will (and is) bloating logs every time when file is directory (happens thousand times on even middle-size project)
the only logging (might) have sense in case of `== nul` or `!isValid`